### PR TITLE
chore: update @oat-sa/tao-item-runner-qti to 2.3.0

### PR DESCRIPTION
**Pull request by taopkorczak**
Related to: https://oat-sa.atlassian.net/browse/TR-6116
Release TR-6116

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -9,6 +9,6 @@
     },
     "dependencies": {
         "@oat-sa/tao-item-runner": "^1.0.0",
-        "@oat-sa/tao-item-runner-qti": "^2.2.3"
+        "@oat-sa/tao-item-runner-qti": "^2.3.0"
```
Comment (olga-kulish) [inline views/package.json:12]: This is backport, so you'll need to release it *manually*, not with CI action.
https://oat-sa.atlassian.net/wiki/spaces/OAT/pages/114885777/How+to+backport
So you'll need to include in this PR:
- version bump for extension itself
- bundle css & js

Comment (taopkorczak) [reply views/package.json:12]: Thanks, done

File: views/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-qti-item",
-    "version": "0.4.0",
+    "version": "0.4.1",
```
Comment (olga-kulish) [inline views/package.json:3]: Sorry, my mistake. No need to version bump extension, please revert it.

But bundles should be here, yes.

Comment (taopkorczak) [reply views/package.json]: Done

**Pull request merged**